src/classes/Country.cls

```diff
@@ -2,26 +2,24 @@
 * Description   - Country Map  
 * Created by    - Sagar Nirwal
 * Created Date  - 30/06/2022
-* Revision log  -  V_1.0 - Created  - Youe name - Date
-*                  V_1.1 - Modified - Youe Name - Date - Details what you modified
+* Revision log  -  V_1.0 - Created  - Sagar Nirwal - 03/07/2022
+*                  V_1.1 - Modified - Sagar NIrwal - 03/07/2022 - Method(getCountryWithLetter) 
 **/
 public with sharing class Country {
 
     //Nested Map using Country with code to get the value countries,states,cites name 
-    Static Map<String,Map<String, Map<String, List<String>>>> countryWithCodeMap = new Map<String,Map<String, Map<String, List<String>>>>();
```

Comment:
I can't find any place where you are putting value in the countryWithCodeMap map.